### PR TITLE
Bug 1720758: jsonnet: make node_exporter pods tolerate every taint

### PR DESCRIPTION
**Pull request by paulfantom**
/cc @brancz @metalmatze @s-urbaniak @squat

### CODE REVIEW

File: jsonnet/node-exporter.jsonnet [hunk 1]
```diff
@@ -104,6 +104,9 @@ local tlsVolumeName = 'node-exporter-tls';
               volumes+: [volume.fromSecret(tlsVolumeName, 'node-exporter-tls')],
               securityContext: {},
               priorityClassName: 'system-cluster-critical',
+              tolerations: [
+                { operator: "Exists" },
```
Comment (brancz) [inline jsonnet/node-exporter.jsonnet]: this should be even in kube-prometheus no?

Comment (paulfantom) [reply jsonnet/node-exporter.jsonnet]: That makes sense. Here's a PR: https://github.com/coreos/kube-prometheus/pull/126

**Pull request merged**